### PR TITLE
SLDS related changes

### PR DESCRIPTION
**Pull request by satyasekharcvb**
### What does this PR do?
Apply SLDS best practices to components

### What issues does this PR fix or reference?
To reflect UI development best practices the sample app needs to make use of SLDS design tokens that make use of the UI density settings in an org. Similar to that any manual CSS should be replaced, where it makes sense, with SLDS classes instead.

#264 

## The PR fulfills these requirements:

[X] Tests for the proposed changes have been added/updated.
[X] Code linting and formatting was performed.

### Functionality Before

NA

### Functionality After

NA

### CODE REVIEW

File: force-app/main/default/aura/auraEmbeddedLWC/auraEmbeddedLWC.css [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
+    width: token(squareIconLargeBoundaryAlt);
+    height: token(squareIconLargeBoundaryAlt);
+    border-radius: token(borderRadiusCircle);
+    margin-bottom: token(varSpacingXSmall);
 }
 
 .THIS .slds-card__footer {
-    text-align: left;
-    color: #706e6b;
+    color: rgb(112, 110, 107);
```
Comment (pchittum) [inline force-app/main/default/aura/auraEmbeddedLWC/auraEmbeddedLWC.css:9]: @satyasekharcvb it was pointed out to me by @pozil that we shouldn't override slds styles. 

There's actually an existing utility class for this color that can be applied right to markup: `slds-color__text_gray-9`. See [here](https://github.com/trailheadapps/lwc-recipes/blob/db1d23f9064538eae2395c8ca9fa75aeada13256/force-app/main/default/aura/lmsPublisherAuraComponent/lmsPublisherAuraComponent.cmp#L38).

In this case, these aura components are marked to be removed from the project anyway in #287, so don't worry about fixing. I just thought you'd like to know.

File: force-app/main/default/aura/defaultTokens/defaultTokens.tokens [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<aura:tokens extends="force:base">
```
Comment (pchittum) [inline force-app/main/default/aura/defaultTokens/defaultTokens.tokens:1]: Is this to simply make sure we have the tokens bundle in the project?

File: force-app/main/default/aura/auraEmbeddedLWC/auraEmbeddedLWC.css (hunk 1 above)
Comment (satyasekharcvb) [reply force-app/main/default/aura/auraEmbeddedLWC/auraEmbeddedLWC.css:9]: For all the LWC, I have used only the design tokens which are GA and the SLDS utility classes that are available in the latest SLDS release notes.

File: force-app/main/default/aura/defaultTokens/defaultTokens.tokens (hunk 2 above)
Comment (satyasekharcvb) [reply force-app/main/default/aura/defaultTokens/defaultTokens.tokens:1]: yes please, so that we can use standard tokens in Aura.

**Pull request merged**